charts/pipelines-library/values.yaml

```diff
@@ -53,6 +53,11 @@ kaniko:
   # -- Save cert in secret "custom-ca-certificates" with key ca.crt
   customCert: false
 
+githubOwners:
```

Comment:
please add description here about this capability and also provide link to official documentation